src/modules/google-organization-kp/includes/Publisher_Service.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+/**
+ * Module:  Google Organization Knowledge Panel
+ * Class:   Publisher_Service
+ *
+ * A class that sets and sets the data for the Google Organization Panel API endpoints.
+ *
+ * @package Wordlift/modules/google-organization-kp
+ *
+ * @since 3.53.0
+ */
+
+namespace Wordlift\Modules\Google_Organization_Kp;
+
+use PHPUnit\Util\Exception;
+
+class Publisher_Service {
+	/**
+	 * @var \Wordlift_Publisher_Service
+	 */
+	private $publisher_service;
+
+	/**
+	 * @var \Wordlift_Entity_Type_Service
+	 */
+	private $entity_service;
+
+	/**
+	 * @var Wordlift_Configuration_Service
+	 */
+	private $configuration_service;
+
+	/**
+	 * @var \Wordlift_Schema_Service
+	 */
+	private $schema_service;
+
+	/**
+	 * @param \Wordlift_Publisher_Service     $publisher_service
+	 * @param \Wordlift_Entity_Type_Service   $entity_service
+	 * @param \Wordlift_Configuration_Service $configuration_service
+	 * @param \Wordlift_Schema_Service        $schema_service
+	 */
+	public function __construct(
+		$publisher_service,
+		$entity_service,
+		$configuration_service,
+		$schema_service
+	) {
+		$this->publisher_service     = $publisher_service;
+		$this->entity_service        = $entity_service;
+		$this->configuration_service = $configuration_service;
+		$this->schema_service        = $schema_service;
+	}
+
+	/**
+	 * Get relevant useful data if a publisher already exists
+	 *
+	 * @return array Publisher data.
+	 *
+	 * @since 3.53.0
+	 */
+	public function get() {
+		$publisher_id = $this->configuration_service->get_publisher_id();
+		if ( ! isset( $publisher_id ) || ! is_numeric( $publisher_id ) ) {
+			return array();
+		}
+
+		$publisher_post = get_post( $publisher_id );
+		if ( ! $publisher_post ) {
+			return array();
+		}
+
+		// Add the publisher fields.
+
+		$data = array();
+
+		$schema_service   = $this->schema_service;
+		$publisher_entity = $this->entity_service->get( $publisher_id );
+		$publisher_logo   = $this->publisher_service->get_publisher_logo( $publisher_id );
+
+		$data['id']          = $publisher_id;
+		$data['name']        = $publisher_post->post_title;
+		$data['type']        = $publisher_entity['label'];
+		$data['description'] = $publisher_entity['description'];
+		$data['alt_name']    = $this->configuration_service->get_alternate_name();
+		$data['legal_name']  = get_post_meta( $publisher_id, $schema_service::FIELD_LEGAL_NAME, true );
+
+		if ( ! empty( $publisher_logo ) ) {
+			$data['image'] = $publisher_logo['url'];
+		}
+
+		$data['same_as']       = get_post_meta( $publisher_id, $schema_service::FIELD_SAME_AS, false );
```

Comment:
`$schema_service::FIELD_SAME_AS` is wrong. You're accessing a static field, use the class name, e.g.

`Wordlift_Scherma_Service::FIELD_SAME_AS`